src/models/supplemental_accessors.jl

```diff
@@ -110,6 +110,14 @@ function get_max_active_power(d::StandardLoad)
     return total_load
 end
 
+function get_max_active_power(d::InterruptibleStandardLoad)
+    total_load = get_max_constant_active_power(d)
+    # TODO: consider voltage
```

Comment:
this is hard one. Lets open an issue to track it since it depends on the modeling assumptions.